src/applications/vaos/services/appointment/transformers.js

```diff
@@ -0,0 +1,157 @@
+import moment from '../../utils/moment-tz';
+
+import { APPOINTMENT_STATUS, APPOINTMENT_TYPES } from '../../utils/constants';
+import { getTimezoneBySystemId } from '../../utils/timezone';
+
+/**
+ * Determines what type of appointment a VAR appointment object is depending on
+ * the existence of certain fields
+ *
+ * @param {Object} appt VAR appointment object
+ */
+function getAppointmentType(appt) {
```

Comment:
Copied several of the appointment util functions into here.  We can remove them from `appointment.js` once we update the application to use FHIR for appointments.